docs/installation/other_inst_scenarios/install_epinio_on_rancher_desktop.md

```diff
@@ -39,6 +42,7 @@ Rancher Desktop configures its own load-balancer to expose Traefik on `127.0.0.1
 
 The Epinio installation is similar on Linux, MacOS and Windows:
 
+<!--TODO: Any particular reason why the Windows sheel prefers the latest?-->
```

Comment:
Looking through the history this part was written by @ldevulder .

Reply:
Thanks.